Fix package.json

```txt
#7 844.5 npm ERR! Could not resolve dependency:
#7 844.5 npm ERR! peer @angular/compiler-cli@"^19.0.0" from @angular-devkit/build-angular@19.1.7
#7 844.5 npm ERR! node_modules/@angular-devkit/build-angular
#7 844.5 npm ERR!   dev @angular-devkit/build-angular@"^19.1.7" from the root project
```

--- a/container-conf/slicops-demo.sh
+++ b/container-conf/slicops-demo.sh
@@ -23,4 +23,7 @@ sleep 2
 Point your browser to http://localhost:$p
 To stop the server, type control-C here.
 EOF
-SLICOPS_CONFIG_UI_API_TCP_IP=0.0.0.0 slicops service ui_api --prod --tcp-port="$p"
+if [[ ! ${APPTAINER_NAME:-} ]]; then
+    export SLICOPS_CONFIG_UI_API_TCP_IP=0.0.0.0
+fi
+slicops service ui_api --prod --tcp-port="$p"

--- a/container-conf/slicops-demo.sh
+++ b/container-conf/slicops-demo.sh
@@ -23,4 +23,4 @@ sleep 2
 Point your browser to http://localhost:$p
 To stop the server, type control-C here.
 EOF
-slicops service ui_api --prod --tcp-port="$p"
+SLICOPS_CONFIG_UI_API_TCP_IP=0.0.0.0 slicops service ui_api --prod --tcp-port="$p"

